src/actions/useActions.ts

```diff
@@ -0,0 +1,44 @@
+import {useClient} from '../hooks/useClient';
+import {useGlobalStore} from '../hooks/useGlobalStore';
+import {CommentData, EntityPointer, SectionData} from '../types';
+
+type UseActionsHookValue = {
+    addCommentToEntity(text: string, entityPointer: EntityPointer): Promise<CommentData>;
+    updateSection(sectionId: string, section: SectionData): Promise<SectionData>;
+}
+
+export const useActions = (): UseActionsHookValue => {
+    const globalStore = useGlobalStore();
+    const client = useClient();
+
+    // TODO: this should be retrieved from globalState
+    // const projectId = globalStore.getCurrentProjectId()
+    const projectId = 'project-1';
+
+    const addCommentToEntity = async (message: string, entityPointer: EntityPointer) => {
```

Comment:
Here `entityPointer` is essentially the thing that the comment is going to be attached to. At the moment, comments can technically be attached to the following:

- project
- section
- file

So if this comment is going to be added to `project-1`, then `entityPointer` would have this value:

```js
{
    entityType: 'project',
    entityId: 'project-1'
}
```